http/proxy/server.go

```diff
@@ -152,6 +161,14 @@ func (server *Server) verifyAccess(ctx echo.Context, request *http.Request, toke
 		if route.operation != "read" {
 			return fmt.Errorf("incorrect operation %s on: %s, must be read", route.operation, serverTaskPath)
 		}
+
+		ctx.SetRequest(patchURL(ctx.Request(), func(requestURL *url.URL) {
+			query := requestURL.Query()
+			query.Set("owner", fmt.Sprintf("Organization/%s", *token.Sub))
```

Comment:
How does this work? I believe our current task has the owner defines as:
```
"owner": {
  "identifier": {
    "system": "http://nuts.nl",
    "value": "did:nuts:UACrv4m3N2kLVjydJ9pe4o3Ytj3Bo88shbBNC3XnNFg"
  }
},
```

Reply:
If I understand it correctly the `owner` field in the task search endpoint should be a reference which is defined as:

```
A reference parameter refers to references between resources. For example, find all Conditions where the subject reference is a particular patient, where the patient is selected by name or identifier. The interpretation of a reference parameter is either:

[parameter]=[id] the logical [id] of a resource using a local reference (i.e. a relative reference)
[parameter]=[type]/[id] the logical [id] of a resource of a specified type using a local reference (i.e. a relative reference), for when the reference can point to different types of resources (e.g. Observation.subject)
[parameter]=[url] where the [url] is an absolute URL - a reference to a resource by its absolute location
```

So, this uses the `[type]/[id]` format where `type` is `Organization` and the `id` the Nuts DID.

Reply:
Note that Nuts identifiers as `owner` are not according to the Nictiz spec (according to @woutslakhorst), because an URI/AGB/UZOVI identifier is expected. Although the spec allows other schemes:

> Definition
The organization’s identification number. For Dutch healthcare providers, the URA number or the AGB number is used, if possible. For foreign or non-affiliated healthcare providers, another unique identification number can be used. This must be accompanied by the name and/or code of the issuing organization.

(https://simplifier.net/packages/nictiz.fhir.nl.stu3.zib2017/2.1.1/files/362362)